File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    minX = len(wallpaper[0]) - 1
    minY = len(wallpaper) - 1
    maxX, maxY = 0,0
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == '#':
                if minX >= j: minX = j      
                if minY >= i: minY = i
                if maxX <= j: maxX = j+1
                if maxY <= i: maxY = i+1
    return [minY, minX, maxY, maxX]

File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.js
function solution(wallpaper) {
    const x = [], y = []
    for(let i=0; i < wallpaper.length; i++){
        for(let j=0; j < wallpaper[0].length; j++){
            if(wallpaper[i][j] === '#') {
                x.push(j);
                y.push(i)
            }
        }
    }
    return [Math.min(...y), Math.min(...x), Math.max(...y) + 1, Math.max(...x) + 1]
}